CHANGES.md

```diff
@@ -96,26 +96,3 @@ Please follow the [`CHANGES.md` modification guidelines](https://github.com/dcos
 * Adjust dcos-net (l4lb) to allow for graceful shutdown of connections by changing the VIP backend weight to `0`
   when tasks are unhealthy or enter the `TASK_KILLING` state instead of removing them. (D2IQ-61077)
 * Set "os:linux" attribute for the Linux agents. (D2IQ-67223)
-
-#### Update Marathon to 1.10.6
-
-* Marathon updated to 1.9.136
-
-* /v2/tasks plaintext output in Marathon 1.5 returned container network endpoints in an unusable way (MARATHON-8721)
-
-* Marathon launched too many tasks. (DCOS_OSS-5679)
-
-* Marathon used to omit pod status report with tasks in `TASK_UNKOWN` state. (MARATHON-8710)
-
-* With UnreachableStrategy, setting `expungeAfterSeconds` and `inactiveAfterSeconds` to the same value will cause the
-instance to be expunged immediately; this helps with `GROUP_BY` or `UNIQUE` constraints. (MARATHON-8719)
-
-* Marathon was checking authorization for unrelated apps when performing a kill-and-scale operations; this has been resolved. (MARATHON-8731)
-
-* A race condition would cause Marathon to fail to start properly. (MARATHON-8741)
-
-#### Update Metronome to 0.6.41
-
-* There was a case where regex validation of project ids was ineffecient for certain inputs. The regex has been optimized. (MARATHON-8730)
-
-* Metronome jobs networking is now configurable (MARATHON-8727)
```

Comment:
Aren't some/all of these changes still relevant? For example, https://jira.d2iq.com/browse/MARATHON-8731 is being resolved in DC/OS 2.1.0, so I think it should appear here?

Reply:
I think we need a clear policy about what belongs in the CHANGES, and what doesn't.

I personally feel like including fixes applied to the stable version of DC/OS don't need to be advertised as fixed in "2.1.0", otherwise, where do we draw the line?

Some guidance is needed here.

The Metronome jobs networking is now configurable should be included, this one is novel to DC/OS. I think there were some merge conflicts and lots of confusion ensued when they were resolved :(

Reply:
@timcharper could we provide links here to the appropriate Marathon/Metronome changelogs? That's what we do for Mesos, could consider it here as well. For example, we currently link to this for the Mesos changes: https://github.com/apache/mesos/blob/00466fe6138138ead1ca676df37c3af69104956d/CHANGELOG